### ISSUE TITLE
[CI] linux://kubernetes:operator is failing/flaky on master

### ISSUE THREAD

**Issue by kevin85421** [ray-project/kuberay#1815]
### What happened + What you expected to happen

KubeRay operator tests are flaky on ray/master. See https://flakey-tests.ray.io/ for more details.

**Comment by anyscalesam** [comment]
@kevin85421 can you please take a look at this sometime the remainder of this week?

**Comment by kevin85421** [comment]
I didn’t find anything on https://flakey-tests.ray.io/?owner=kuberay. Does this mean the test is no longer flaky?

**Comment by anyscalesam** [comment]
cc @can-anyscale

**Comment by can-anyscale** [comment]
@kevin85421 yes no longer flaky, it's now show up as under kuberay team (https://flakey-tests.ray.io/?owner=kuberay), but since the test never fails so it's not showing up at all
